### PR TITLE
YugaV39 || Changes made for yuga classifier

### PR DESCRIPTION
**Pull request by sonali1404**

### CODE REVIEW

**Review by vizsatiz — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 1]
```diff
@@ -158,6 +158,11 @@
             <version>${jackson.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin.external.google</groupId>
```
[the remark this answers is not in the document]
Comment (sonali1404) [reply pom.xml]: Removed it as I also didn't find any use.

File: src/main/java/com/twelfthmile/yuga/types/Trie.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            if (!node.hasNext(currentChar)) {
+                node.put(currentChar, new TrieNode());
+            }
+            node = node.get(currentChar);
+        }
+        node.setEnd();
+        node.setLabel(label);
+    }
+    public void insertUpis() {
+        String[] arr = {"airtel", "airtelpaymentsbank", "albk", "allahabadbank", "allbank", "andb", "apb", "apl", "axis", "axisb", "axisbank",
```
[the remark this answers is not in the document]
Comment (sonali1404) [reply src/main/java/com/twelfthmile/yuga/types/Trie.java]: Done.

File: src/main/java/com/twelfthmile/yuga/types/Trie.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                "okhdfcbank", "okicici", "oksbi", "paytm", "payzapp", "pingpay", "pockets", "pnb", "psb", "purz", "rajgovhdfcbank", "sbi",
+                "sc", "scb", "scbl", "scmobile", "sib", "srcb", "synd", "syndbank", "syndicate", "tjsb", "ubi", "uboi",
+                "uco", "unionbank", "unionbankofindia", "united", "utbi", "vijayabank", "vijb", "vjb", "ybl", "yesbank", "yesbankltd", "upi", "ibl"};
+        String label = "UPI";
+        for(int i = 0; i < arr.length; i++) {
+            insert(arr[i], label);
+        }
+    }
+    private void insertCurr() {
+        String[] arr = {"rs", "inr", "cny", "amt", "amount", "ngn", "usd", "cad", "eur", "gbp", "aed", "jpy", "aud", "s$", "lkr", "ksh", "egp"};
```
[the remark this answers is not in the document]
Comment (sonali1404) [reply src/main/java/com/twelfthmile/yuga/types/Trie.java]: Done

File: src/main/java/com/twelfthmile/yuga/types/Pair.java [hunk 4]
```diff
@@ -8,8 +8,8 @@
 
 public class Pair<A, B> {
 
-    private final A a;
-    private final B b;
+    A a;
+    B b;
```
[the remark this answers is not in the document]
Comment (sonali1404) [reply src/main/java/com/twelfthmile/yuga/types/Pair.java]: Done

File: src/main/java/com/twelfthmile/yuga/types/Trie.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    }
+    private void insertInstr() {
+        String[] arr = {"card", "no", "a/c"};
+        String label = "INSTR";
+        for(int i = 0; i < arr.length; i++) {
+            insert(arr[i], label);
+        }
+    }
+    private void insertFltId() {
+        String[] arr = {"6e", "indigo", "ai", "airindia", "sg", "spicejet", "g8", "goair", "uk", "vistara", "ix",
```
[the remark this answers is not in the document]
Comment (sonali1404) [reply src/main/java/com/twelfthmile/yuga/types/Trie.java]: Done

**Review by vizsatiz — APPROVED** [review]

**Pull request merged**